code/modules/surgery/bodyparts/bodyparts.dm

```diff
@@ -0,0 +1,337 @@
+	else
+		skin_tone = ""
+
+	body_gender = H.gender
+	should_draw_gender = S.sexes
+
+	if(MUTCOLORS in S.specflags)
+		species_color = H.dna.features["mcolor"]
+		should_draw_greyscale = TRUE
+	else
+		species_color = ""
+
+	if(H.disabilities & HUSK)
+		species_id = "husk"
+		should_draw_gender = FALSE
+		should_draw_greyscale = FALSE
+
+	if(!dropping_limb && H.dna.check_mutation(HULK))
+		mutation_color = "00aa00"
+	else
+		mutation_color = ""
+
+	if(dropping_limb)
+		no_update = 1 //when attached, the limb won't be affected by the appearance changes of its mob owner.
+
+//to update the bodypart's icon when not attached to a mob
+/obj/item/bodypart/proc/update_icon_dropped()
+	overlays.Cut()
+	var/image/I = get_limb_icon(1)
+	if(I)
+		I.pixel_x = px_x
+		I.pixel_y = px_y
+		overlays += I
+
+//Gives you a proper icon appearance for the dismembered limb
+/obj/item/bodypart/proc/get_limb_icon(dropped)
+	var/image/I
+
+	var/icon_gender = (body_gender == FEMALE) ? "f" : "m" //gender of the icon, if applicable
+
+	if((body_zone != "head" && body_zone != "chest"))
+		should_draw_gender = FALSE
+
+	var/image_dir
+	if(dropped)
+		image_dir = SOUTH
+	if(status == ORGAN_ORGANIC)
+		if(should_draw_greyscale)
+			if(should_draw_gender)
+				I = image("icon"='icons/mob/human_parts_greyscale.dmi', "icon_state"="[species_id]_[body_zone]_[icon_gender]_s", "layer"=-BODYPARTS_LAYER, "dir"=image_dir)
+			else
+				I = image("icon"='icons/mob/human_parts_greyscale.dmi', "icon_state"="[species_id]_[body_zone]_s", "layer"=-BODYPARTS_LAYER, "dir"=image_dir)
+		else
+			if(should_draw_gender)
+				I = image("icon"='icons/mob/human_parts.dmi', "icon_state"="[species_id]_[body_zone]_[icon_gender]_s", "layer"=-BODYPARTS_LAYER, "dir"=image_dir)
+			else
+				I = image("icon"='icons/mob/human_parts.dmi', "icon_state"="[species_id]_[body_zone]_s", "layer"=-BODYPARTS_LAYER, "dir"=image_dir)
+	else
+		if(should_draw_gender)
+			I = image("icon"='icons/mob/augments.dmi', "icon_state"="[body_zone]_[icon_gender]_s", "layer"=-BODYPARTS_LAYER, "dir"=image_dir)
+		else
+			I = image("icon"='icons/mob/augments.dmi', "icon_state"="[body_zone]_s", "layer"=-BODYPARTS_LAYER, "dir"=image_dir)
+		return I
+
+
+	if(!should_draw_greyscale)
+		return I
+
+	//Greyscale Colouring
+	var/draw_color
+
+	if(skin_tone) //Limb has skin color variable defined, use it
+		draw_color = skintone2hex(skin_tone)
+	if(species_color)
+		draw_color = species_color
+	if(mutation_color)
+		draw_color = mutation_color
+
+	if(draw_color)
+		I.color = "#[draw_color]"
+	//End Greyscale Colouring
+
+	return I
+
+
+
+/obj/item/bodypart/chest
+	name = "chest"
+	desc = "why is it detached..."
+	icon_state = "chest"
+	max_damage = 200
+	body_zone = "chest"
+	body_part = CHEST
+	px_x = 0
+	px_y = 0
+	var/obj/item/cavity_item
+
+/obj/item/bodypart/chest/Destroy()
+	if(cavity_item)
+		qdel(cavity_item)
+	return ..()
+
+/obj/item/bodypart/l_arm
+	name = "left arm"
+	desc = "why is it detached..."
```

Comment:
I don't like that one it's too wordy
If we need a witty description for the hands/arms just say something like "This reminds you of your girlfriend."